### PR TITLE
Fix some grammatical errors and typos in README.MD and CONTRIBUTING.MD

### PR DESCRIPTION
**Pull request by pontakornth**
- Add commas in "if" statement
- Fix some typos in both files
- Make Note and TL;DR more visible by using **bold font.**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,10 +55,10 @@ We also welcome pull request. The [issue tracker](https://github.com/Rurusetto/r
...
+  Even GitHub provide an option to edit code or replace files in the repository using the web interface. We strongly discourage using it in most scenarios. Editing files this way is inefficient and likely to introduce whitespace or file encoding changes that make it more difficult to review the code.
 
   Code written in web interface will also very likely be questioned by the reviewer or fail the continuous integration and code quality check. We strongly encourage using an IDE like [PyCharm](https://www.jetbrains.com/pycharm/) or [Visual Studio Code](https://code.visualstudio.com/).
 
 * **Feel free to reach us for help.**
 
-  If you have any problem on the development process please leave the comment in your pull request or contact the maintainer in the [Discord development server](https://discord.gg/CQPNADu). We will help you as much as we can.
+  If you have any problem on the development proces, please leave the comment in your pull request or contact the maintainer in the [Discord development server](https://discord.gg/CQPNADu). We will help you as much as we can.
```
Comment (HelloYeew) [inline CONTRIBUTING.md]:
```suggestion
  If you have any problem on the development process, please leave the comment in your pull request or contact the maintainer in the [Discord development server](https://discord.gg/CQPNADu). We will help you as much as we can.
```
A little bit typo here

**Review by HelloYeew — CHANGES_REQUESTED** [review]
A little bit fix to go!

File: README.md [hunk 2]
```diff
@@ -120,15 +120,17 @@ python manage.py runserver
...
 
 We use [code of conduct](code_of_conduct.md) from `Contributor Covenant`.
 
 ## License
 
-The code in this repository is licensed under MIT license. Please see [the license file](LICENSE) for more information. tl;dr you can do whatever you want as long as you include the original copyright and license notice in any copy of the software/source.
+The code in this repository is licensed under MIT license. Please see [the license file](LICENSE) for more information. 
 
-The image in this website and the content that upload by user and community is cover on DMCA. If you are the work and owner and want to request to remove your work please email the team at [contact@rulesets.info](mailto:contact@rulesets.info) or contact HelloYeew (project leader) at [me@helloyeew.dev](mailto:me@helloyeew.dev).
+TL;DR** you can do whatever you want as long as you include the original copyright and license notice in any copy of the software/source.
```
Comment (HelloYeew) [inline README.md]:
```suggestion
**TL;DR** you can do whatever you want as long as you include the original copyright and license notice in any copy of the software/source.
```

I just notice it's not fully bold.

**Review by HelloYeew — CHANGES_REQUESTED** [review]
A little bit more (I just notice it.(

**Review by HelloYeew — APPROVED** [review]

**Comment by HelloYeew** [comment]
Thanks for the contribution!
